frontend/src/metabase/dashboard/components/DashCard/DashCardMenu/DashCardMenuItems.tsx

```diff
@@ -99,7 +99,7 @@ export const DashCardMenuItems = ({
       fw="bold"
       {...item}
       key={item.key}
-      icon={<Icon name={item.iconName} />}
+      icon={<Icon name={item.iconName} aria-hidden />}
```

Comment:
This was to make sure we could target the dropdown menu with label, without this change there will be the icon name in the label.

I, however, remove those tests at the end because it wasn't working with loki for some reasons.